bindata/manifests/switchdev-config/files/bind-switchdev.sh.yaml

```diff
@@ -0,0 +1,35 @@
+mode: 0755
+overwrite: true
+path: "/usr/local/bin/bind-switchdev.sh"
```

Comment:
The name looks unclear to me. Since we split the switchdev-configuration.service into two portion. Shall we use the name to `switchdev-configuration-before-NM.service` and `switchdev-configuration-after-NM.service`.

Reply:
Done